src/test/java/org/junitpioneer/jupiter/DefaultTimeZoneTests.java

```diff
@@ -35,16 +34,15 @@
 
 	@BeforeAll
 	static void globalSetUp() {
-		// the extension sets UTC as test time zone unless it is already
-		// the system's time zone; in that case it uses GMT
+		// we set UTC as test time zone unless it is already
+		// the system's time zone; in that case we use UTC+12
 		DEFAULT_TIMEZONE_BEFORE_TEST = TimeZone.getDefault();
 		TimeZone utc = TimeZone.getTimeZone("UTC");
-		TimeZone gmt = TimeZone.getTimeZone("GMT");
-		if (DEFAULT_TIMEZONE_BEFORE_TEST.equals(utc)) {
-			TimeZone.setDefault(gmt);
-		} else {
+		TimeZone utcPlusTwelve = TimeZone.getTimeZone("GMT+12:00");
```

Comment:
@nicolaiparlog Shouldn't this technically be "UTC+12:00" if you want UTC+12? There can be a tiny difference between UTC and GMT if I remember correctly. Probably not relevant for the tests here, but I thought I mention it anyway.

Here's a video by Tom Scott that mentions said difference: https://www.youtube.com/watch?v=h0AMaW4XRCI
(It's entirely possible that I'm misinterpreting the things said in the video though...)

Reply:
```
jshell> TimeZone.getTimeZone("UTC+12:00")
$1 ==> sun.util.calendar.ZoneInfo[id="GMT",offset=0,dstSavings=0,useDaylight=false,transitions=0,lastRule=null]
```

:man_shrugging: 

That's why I picked `TimeZone.getTimeZone("GMT+12:00")` instead.

Reply:
I see. That's odd. But at least you tried :sweat_smile:

Reply:
Looks like UTC is technically not a timezone, which is why `TimeZone.getTimeZone("UTC+12:00")` will return GMT as a default instead. From https://www.timeanddate.com/time/gmt-utc-time.html:

> UTC is not a time zone, but a time standard that is the basis for civil time and time zones worldwide. This means that no country or territory officially uses UTC as a local time.

Today I learned.